core/src/main/resources/bootstrap/tuningTable.yaml

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+tuningDefinitions:
+  - label: spark.executor.cores
+    description: ''
```

Comment:
can you file a follow-on issue to popualte descriptions for all of these settings?  feel free to assign to me and i can do it next week

Reply:
I populated the descriptions and some of the spark-defaults. We can later re-iterate to improve and expand to add fields such as `deprecated`..etc.